nightingale_interface/screens/item_select_screen.py

```diff
@@ -0,0 +1,273 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+from kivy.app import App
+
+
+class ItemSelectScreen:
+    # below are the idx for the labels in the children array
+    water_count_label_idx = 2
+    ice_count_label_idx = 1
+    blanket_count_label_idx = 0
+
+    # screen id from the screen manager
+    item_select_screen_id = 9
+
+    def on_enter(self, *args):
+        print("CALLED")
+        self.reset_counts()
+
+    def add_item(self, button_data):
+        if button_data.id == "water" and self.water_count < 3:
+            self.water_count += 1
+        if button_data.id == "ice" and self.ice_count < 3:
+            self.ice_count += 1
+        if button_data.id == "blanket" and self.blanket_count < 3:
+            self.blanket_count += 1
+
+    def remove_item(self, button_data):
+        if button_data.id == "water" and self.water_count > 0:
+            self.water_count -= 1
+        if button_data.id == "ice" and self.ice_count > 0:
+            self.ice_count -= 1
+        if button_data.id == "blanket" and self.blanket_count > 0:
+            self.blanket_count -= 1
+
+    # functions to update text on count change
+    def on_water_count(self, *args):
```

Comment:
Will need to add functionality to increment counters on item fill screen at same time